parltrack_meps/management/commands/update_meps.py

```diff
@@ -70,12 +73,14 @@ def handle(self, *args, **options):
         print "unxz dump"
         os.system("unxz %s" % JSON_DUMP_ARCHIVE_LOCALIZATION)
         print "load json"
-        meps = json.load(open(JSON_DUMP_LOCALIZATION, "r"))
+        #meps = json.load(open(JSON_DUMP_LOCALIZATION, "r"))
```

Comment:
It's a good practice to leave a comment about why this code is commented, else better to choose which like to commit ;)